proxy/http/HttpSM.cc

```diff
@@ -6339,9 +6330,9 @@ HttpSM::find_http_resp_buffer_size(int64_t content_length)
     }
   } else {
 #ifdef WRITE_AND_TRANSFER
-    buf_size = HTTP_HEADER_BUFFER_SIZE + content_length - index_to_buffer_size(HTTP_SERVER_RESP_HDR_BUFFER_INDEX);
+    int64_t buf_size = HTTP_HEADER_BUFFER_SIZE + content_length - index_to_buffer_size(HTTP_SERVER_RESP_HDR_BUFFER_INDEX);
```

Comment:
I would not do this unless the variable is used only in an ifdef scope, because it can easily make code uncompilable.

Reply:
I can move out the declaration outside the ifdef if you prefer.